api/src/main/java/org/apache/cloudstack/api/command/user/config/ListCapabilitiesCmd.java

```diff
@@ -59,6 +59,15 @@ public void execute() {
         response.setKVMSnapshotEnabled((Boolean)capabilities.get("KVMSnapshotEnabled"));
         response.setAllowUserViewDestroyedVM((Boolean)capabilities.get("allowUserViewDestroyedVM"));
         response.setAllowUserExpungeRecoverVM((Boolean)capabilities.get("allowUserExpungeRecoverVM"));
+
+        response.setVmSnapshotsMax((Integer)capabilities.get("vmsnapshotsmax"));
```

Comment:
What about extracting these names to enum? and then documenting the option/names to better explain what they mean.